spec/models/medicaid_application_attributes_spec.rb

```diff
@@ -14,7 +26,48 @@
 
       expect(data).to eq(
         signature: "Jane Hancock",
+        mailing_address_city: "Flint",
+        mailing_address_county: "Genesee",
+        mailing_address_state: "MI",
+        mailing_address_street_address: "123 Main St.",
+        mailing_address_zip: "12345",
+        residential_address_city: "Flint",
+        residential_address_county: "Genesee",
+        residential_address_state: "MI",
+        residential_address_street_address: "456 I Live Here Ave",
+        residential_address_zip: "12345",
+        phone_number_0: "0",
```

Comment:
note that I am not testing that these `phone_` fields are filled out properly in the actual PDF...but I could. Not sure how much to test there. Here we are confirming that the hash is formed properly but we will not know that the PDF fields are named properly for sure unless we test everything in the `Dhs1426Pdf` class...

Reply:
I feel like we should move toward testing everything in the PDF classes, since that's the only time we pull it out of the PDF and that's the important part. I feel like this is more important to test the translation of application attributes to fields (if there's any logic), but realize that leaves the tests pretty thin.

basically, I'm not a big fan of duplicating these attribute tests, but also feel like right now we're not thoroughly testing the most important part.